index.js

```diff
@@ -831,12 +839,17 @@
 
     var padding = _(showTypeSig_(expTypes.slice(0, index)));
 
+    //  f :: (String -> String) -> String
+    var f = function(f) {
+      return trimTrailingSpaces(underline(typePath[0], propPath, f));
+    };
+
```

Comment:
While updating Sanctuary I noticed that I'd forgotten to apply `trimTrailingSpaces` below. I updated the definition of `f` accordingly. There's now no reason to curry `underline`, so I reverted that change.